### PR TITLE
karabiner.rb: Upgrade to 10.7.0

### PR DESCRIPTION
**Pull request by xight**

### CODE REVIEW

File: Casks/karabiner.rb [hunk 1]
```diff
@@ -4,13 +4,13 @@
...
   appcast 'https://pqrs.org/osx/karabiner/files/appcast.xml',
-          :sha256 => '5be44fb038e6b4842367e5186c9e2a366117e49691c778367dbb5d92dfb97bd5'
+          :sha256 => '309c41f2569783084f4e9bf7085d2749a482fe3d3f2ca58bf9be0da89c2956a1'
   name 'Karabiner'
   name 'KeyRemap4MacBook'
   homepage 'https://pqrs.org/osx/karabiner/'
   license :public_domain
 
-  pkg 'Karabiner.pkg'
+  pkg 'Karabiner.sparkle_guided.pkg'
```
Comment (vitorgalvao) [inline Casks/karabiner.rb]: Again, where are you getting this from?

Comment (xight) [inline Casks/karabiner.rb]: I got the file from this URL.
https://pqrs.org/osx/karabiner/files/Karabiner-10.7.0.dmg

I found that pkg name was changed. so I fixed a pkg name.

Comment (vitorgalvao) [inline Casks/karabiner.rb]: Ah, I had actually checked (hence the question) but just saw I checked the old version.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It was merged directly as 013eeea5c06f3e99bcfae5ce2359e3847862cb6d to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**